tests/unit/design_document_tests.py

```diff
@@ -754,6 +756,22 @@ def test_get_info(self):
              'name': name
             })
 
+    def test_get_info_raises_httperror(self):
+        """
+        Test get_info raises an HTTPError.
+        """
+        # Mock HTTPError when running against CouchDB and Cloudant
+        resp = requests.Response()
+        resp.status_code = 400
+        self.client.r_session.get = mock.Mock(return_value=resp)
+        with self.assertRaises(requests.HTTPError) as cm:
+            ddoc = DesignDocument(self.db, '_design/ddoc001')
```

Comment:
I would move this line before the `with self.assertRaises...` that way it ensures that the HTTPError is being raised on the call to `.info()`.